src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -84,6 +83,55 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    //ToDo(kristenmason@): implement the Google Maps request access functionality
+    this.simidProtocol.sendMessage(CreativeMessage.REQUEST_PAUSE).then(() => {
+      findNearest.classList.add("hidden");
```

Comment:
If you're hiding the button either way, might as well hide if before sending the message so you only have to do it once

Reply:
true, fixed.